app/resources/api/v2/barcode_printer_resource.rb

```diff
@@ -13,25 +23,26 @@ class BarcodePrinterResource < BaseResource
       # Attributes
       ###
 
-      # @!attribute [r]
-      # @return [String] The name of the barcode printer.
+      # @!attribute [r] name
+      #   @return [String] the name of the barcode printer.
       attribute :name
```

Comment:
Does this need to be `readonly: true` to align it with the others since the resource is immutable? I appreciate its not part of the documentation so feel free to ignore.

Reply:
I found it hard to know what to put in the documentation for immutable resources.  On the one hand, you could write to these attributes because they aren't marked as `readonly: true` but on the other hand, you can't write to the endpoint at all.  I think it doesn't really matter either way to be honest since it's immutable.  However I don't know that we should be using `readonly: true` at all -- see the other PR.  It's definitely not a documentation concern though.